torch_xla/distributed/xla_dist.py

```diff
@@ -86,14 +95,38 @@ def __init__(self,
                ' will interfere with the values set for distributed'
                ' training'.format(dist_var)))
 
+  def _check_client_mesh_health(self):
+
+    def _pop_mark_step_queue(queue, timeout=30):
+      queue.get(timeout=timeout)
+
+    with futures.ThreadPoolExecutor(
+        max_workers=len(self._heartbeats)) as executor:
+      try:
+        results = executor.map(_pop_mark_step_queue, self._heartbeats.values())
+        for result in results:
+          if result:
+            result.result()  # Only iterating to re-raise any worker errors.
+      except queue.Empty:
+        self._error_queue.put(RuntimeError(
+          'Client mesh is unhealthy due to non-matching heartbeats'))
+
   def _stream_logs(self, process, client_worker):
     client_ip = client_worker._internal_ip
     ordinal = self._cluster._client_workers.index(client_worker)
 
     def _stream_output(stream, log_fn):
       for std in iter(stream.readline, b''):
+        std_line = std.decode('utf-8').rstrip('\n')
+        if 'torch_xla.core.xla_model::mark_step' in std_line:
```

Comment:
can we make this a constant + import instead of relying it's the same value in 2 different files?

Reply:
Good idea, done.